meson.build

```diff
@@ -456,10 +457,10 @@ foreach lib:enabled_pmd_libs
 endforeach
 
 cndp_pmds_a_name = 'libcndp_pmds.a'
-run_command(mklib, cndp_pmds_a_name, libcndp_pmds_a, check: true)
+run_command(mklib, tempdir, cndp_pmds_a_name, libcndp_pmds_a, check: true)
 cndp_pmds_a = custom_target('libcndp_pmds_a_target',
     output: cndp_pmds_a_name,
-    input: join_paths('/tmp', cndp_pmds_a_name),
+    input: join_paths(tempdir, cndp_pmds_a_name),
     command:[find_program('cp'), '@INPUT@', cndp_pmds_a_name],
     install_dir: join_paths('lib', 'x86_64-linux-gnu'),
     install: true)
```

Comment:
We should be able to fix this problem with the following:

```
mklib = find_program('tools/mklib.sh')
build_dir = meson.current_build_dir()

if use_static_libs
    cndp_a = custom_target('libcndp_a_target',
        output: 'libcndp.a',
        command:[mklib, build_dir, '@OUTPUT@', libcndp_a],
        install_dir: join_paths('lib', 'x86_64-linux-gnu'),
        install: true)
else
    cndp_so = custom_target('libcndp_so_target',
        output: 'libcndp.so',
        command:[mklib, build_dir, '@OUTPUT@', libcndp_so],
        install_dir: join_paths('lib', 'x86_64-linux-gnu'),
        install: true)
endif

# Generate libcndp_pmds.a
libcndp_pmds_a = []
foreach lib:enabled_pmd_libs
    libcndp_pmds_a += 'lib' + lib + '.a '
endforeach

cndp_pmds_a = custom_target('libcndp_pmds_a_target',
    output: 'libcndp_pmds.a',
    command:[mklib, build_dir, '@OUTPUT@', libcndp_pmds_a],
    install_dir: join_paths('lib', 'x86_64-linux-gnu'),
    install: true)
```

Building the linker files directly into the build directory to avoid the /tmp conflict.